python/cuml/linear_model/linear_regression.pyx

```diff
@@ -260,7 +260,8 @@ class LinearRegression(LinearPredictMixin,
     intercept_ = CumlArrayDescriptor(order='F')
 
     @device_interop_preparation
-    def __init__(self, *, algorithm='eig', fit_intercept=True, normalize=False,
+    def __init__(self, *, algorithm='eig', fit_intercept=True,
+                 normalize=False, copy_X=True,
```

Comment:
I'd recommend to move the `copy_X` parameter behind `fit_intercept` to more closely match the sklearn API. Since these parameters are key-word only, the order does in principle not matter, but I think it would still be nice to match the API as close as possible.